src/EasyHttp/Http/HttpClient.cs

```diff
@@ -71,15 +71,24 @@ public class HttpClient
         readonly IEncoder _encoder;
         readonly IDecoder _decoder;
         readonly UriComposer _uriComposer;
-        
+        private bool m_ShouldRemoveAtSign = true;
+
```

Comment:
Is it possible to make this simply  _shouldRemoveAtSign = true to follow other conventions?